brokerapi/brokers/cloudsql/broker.go

```diff
@@ -417,18 +417,18 @@ func combineServiceBindingCreds(sqlCreds models.ServiceBindingCredentials, saCre
 }
 
 func (b *CloudSQLBroker) BuildInstanceCredentials(ctx context.Context, bindDetails models.ServiceBindingCredentials, instanceDetails models.ServiceInstanceDetails) (map[string]string, error) {
-	return b.AccountManager.BuildInstanceCredentials(ctx, bindDetails, instanceDetails)
+	return b.sqlAccountManager().BuildInstanceCredentials(ctx, bindDetails, instanceDetails)
 }
 
 // Unbind deletes the database user, service account and invalidates the ssl certs associated with this binding.
 func (b *CloudSQLBroker) Unbind(ctx context.Context, creds models.ServiceBindingCredentials) error {
 	var accumulator error
 
-	if err := b.AccountManager.DeleteCredentials(ctx, creds); err != nil {
+	if err := b.sqlAccountManager().DeleteCredentials(ctx, creds); err != nil {
```

Comment:
Instead of having separate account manager objects and calling DeleteCreds() on both of them, I wonder if it might not be appropriate to take an approach where SqlAccountManager extends AccountManager, overriding DeleteCreds() and calling super.DeleteCreds() OO internally traditional OO style. It might clean up this upstream code a bit. Here's a good article which shows an example of how to do this in Go: https://medium.com/random-go-tips/method-overriding-680cfd51ce40